integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/gcp/action/GcpClientActions.java

```diff
@@ -307,15 +322,8 @@ public void listBucketSelectedObject(String baseLocation, boolean zeroContent) {
             throw new TestFailException(format(" Google GCS path: '%s' does not exist! ", keyPrefix));
         } else {
             for (Blob blob : blobs.iterateAll()) {
-                try {
-                    URI selfLink = new URI(blob.getSelfLink());
-                    if (selfLink.getPath().contains(selectedObjectPath)) {
-                        filteredBlobs.add(blob);
-                    }
-                } catch (URISyntaxException e) {
-                    Log.error(LOGGER, format("Google GCS object: '%s' path: '%s' is not a valid URI!", blob.getName(), blob.getSelfLink()));
-                    throw new TestFailException(format(" Google GCS object: '%s' path: '%s' is not a valid URI!",
-                            blob.getName(), blob.getSelfLink()));
+                if (StringUtils.remove(blob.getName(), "/").contains(StringUtils.remove(selectedObjectPath, "/"))) {
```

Comment:
Here the blob's name and the selected path can look like for example:
- `blob.getName()=pre-termination/gcp-test-19155dbba0/`,
- `selectedObjectPath=/pre-termination/gcp-test-19155dbba0`
or
- `blob.getName()=apitest5089b22866494e35bfb831d79cdc45ad/backups/`
- `selectedObjectPath=/apitest5089b22866494e35bfb831d79cdc45ad/backups`

So I do not think this can cause false result actually.